app/src/Filters.jsx

```diff
@@ -10,7 +10,7 @@ export function Filters() {
   const params = context.params;
   const setParams = context.setParams;
   const filters = getFilterParams(params);
-  const klass = 'filters' + (Object.keys(filters).length ? 'active' : '');
+  const klass = 'filters' + (Object.keys(filters).length ? ' active' : '');
```

Comment:
Was this the only edit you made? Did you run _npm run appbuild_ as well?

Reply:
@UKChris-osm No, I didn't run , but I just hit `npm run appbuild` in local and yes it is working!
also I manually made a change in the `docs/index.js`

Reply:
What manual edits did you make to `index.js`? I'm not sure how that file is generated, but I thought it was all automated, and so manual edits might disappear once `npm run appbuild` is run again.

Reply:
![Screenshot 2023-09-19 203049](https://github.com/osmlab/name-suggestion-index/assets/58364635/7ce0167d-266d-449f-bf52-0651211e5c1c)
this one! 
Yes, you're right it is auto generated but the fix I made before `npm run appbuild` and after totally working fine.
making change in `filters.js` alone was enough as the issue was fixed, but to made sure I also dived into `index.js`.

Reply:
As `npm run appbuild` should generate the desired changes, it's best to leave that file alone, and the change to the filters class should only be needed in the one file, the initial one you edited.

Unfortunately I'm without a computer at the moment, so can't run any of the npm software myself.

What changes were made to the files below, as I've lost track? It might be best to start afresh, and just edit the one line for the filters class, then run `npm run appbuild `and commit that as a PR.

Reply:
I have done what you said in my latest commit. kindly check it